### PR TITLE
Bringing Dockerfile 71 up-to-date with 70

### PR DESCRIPTION
**Pull request by winfried-van-loon**
This is a try to bring the PHP-FPM PHP7 Dockerfiles up-to-date with each other.

### CODE REVIEW

**Comment by mikeerickson** [comment]
@winfried-van-loon Is this the issue which fixes my reported issue of PHP71 not being installed?

**Comment by winfried-van-loon** [comment]
No, these fixes are not related to any issue at the moment. They only merge the existing options in Dockerfile-70 into Dockerfile-71.

**Comment by winfried-van-loon** [comment]
I'm running tests on the merged changes now, will then run tests on this issue's changes.

File: php-fpm/Dockerfile-71 [hunk 1]
```diff
@@ -114,6 +136,7 @@ RUN if [ ${INSTALL_AEROSPIKE_EXTENSION} = true ]; then \
         && make install \
     ) \
     && rm /tmp/aerospike-client-php.tar.gz \
+    && docker-php-ext-enable aerospike \
```
Comment (winfried-van-loon) [inline php-fpm/Dockerfile-71]: This may not be needed

**Comment by winfried-van-loon** [comment]
Okay, this shouldn't lead to additional warnings or errors. (tested)

**Pull request merged**